cabal.project

```diff
@@ -1,10 +1,4 @@
-packages:
-    pact.cabal
-
-package pact
-    tests: True
-
-debug-info: True
```

Comment:
These were unnecessary to have on all the time. I've moved them to my local `cabal.project.local`.